spec/rofl-puzzles_spec.rb

```diff
@@ -0,0 +1,56 @@
+load "scripts/rofl-puzzles.lic"
```

Comment:
AFAIK there is no way to get require to load a `.lic` file. The big difference in behavior is that require will try to be smart and only execute the file once, but load will do it on every invocation. I don't think it matters for our use case, but I haven't used load for much other than this so there may be some other gotchas.

Reply:
The only difference I am aware of between `load` and `require` in this context is that `load` is not cached by the global `$LOADED_FEATURES` variable